modules/self-test/test_helpers.lua

```diff
@@ -44,7 +44,10 @@
 		return prj
 	end
 
-
+	function m.createGroup(wks)		
+		local prj = group ("MyGroup" .. (#wks.groups + 1))
```

Comment:
Technically, we don't need this function at all…you could just use `group()` directly and have full control over the name. But it doesn't hurt to have it for completeness I suppose?

Either way, the change under test is certainly a good one.